class ConnectionError(Exception):
    pass

class ValidationError(Exception):
    pass
